plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/general/ButtonLink.tsx

```diff
@@ -0,0 +1,108 @@
+import { styled } from "@web/theme";
+import { useCallback, useState } from "react";
+
+import { BaseFieldProps, ButtonLinkInfo } from "../types";
+
+const ButtonLink: React.FC<BaseFieldProps<"buttonLink">> = ({ editMode }) => {
+  const [CurrentButton, setCurrentButton] = useState<ButtonLinkInfo>();
+
+  const handleChangeButtonTitle = useCallback((e: React.ChangeEvent<HTMLInputElement>) => {
+    setCurrentButton(btn => {
+      if (!btn) return;
+      return { ...btn, title: e.currentTarget.value };
+    });
+  }, []);
+
+  const handleChangeButtonLink = useCallback((e: React.ChangeEvent<HTMLInputElement>) => {
+    setCurrentButton(btn => {
+      if (!btn) return;
+      return { ...btn, link: e.currentTarget.value };
+    });
+  }, []);
+  return editMode ? (
+    <Wrapper>
+      <Field>
+        <FieldTitle>Title</FieldTitle>
+        <FieldValue>
+          <TextInput onChange={handleChangeButtonTitle} />
+        </FieldValue>
+      </Field>
+
+      <Field>
+        <FieldTitle>Link</FieldTitle>
+        <FieldValue>
+          <TextInput onChange={handleChangeButtonLink} />
+        </FieldValue>
+      </Field>
+    </Wrapper>
+  ) : (
+    <Wrapper>
+      <Field>
+        <FieldTitle />
+        <FieldValue>
+          <StyledButton onClick={() => window.open(CurrentButton?.link, "_blank", "noopener")}>
+            <Text>{CurrentButton?.title}</Text>
+          </StyledButton>
+        </FieldValue>
+      </Field>
+    </Wrapper>
```

Comment:
Only the StyledButton is necessary, right?

Reply:
I will only remove the field title to keep same design

Reply:
30294f5aa82bcbb0ea19d6d665ee117a42cea95a